tests/unit/scripts/test_ar_report.py

```diff
@@ -1,4 +1,4 @@
-from ar_report import *
+from scripts.ar_report import *
```

Comment:
This change was not needed. I thought it was needed because I had a local `pytest.ini` that was picked up instead of the `pytest.ini` section in `pyproject.toml` where `scripts` is added to the python path.